packages/design-system/src/components/N8nSquareButton/SquareButton.vue

```diff
@@ -0,0 +1,48 @@
+<template functional>
+	<button :class="$style.button" @click="(e) => listeners.click && listeners.click(e)">
+		<span :class="$style.text" v-text="props.label" />
+	</button>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+export default Vue.extend({
+	name: 'n8n-square-button',
+	props: {
+		label: {
+			type: String,
+		},
+	},
+});
+</script>
+
+<style lang="scss" module>
+.button {
+	width: 28px;
+	height: 29px;
+	border-radius: 4px;
+	border: var(--color-background-xlight);
+	cursor: pointer;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+
+	@media (max-width: $--breakpoint-xs) {
+		width: 20px;
+		height: 20px;
+	}
```

Comment:
ideally we should be setting width/height through a prop... it does not make sense to have media queries in components like this.

Reply:
Removed.